notebooks/samples/LightGBM Overview.ipynb

```diff
@@ -0,0 +1,548 @@
+  },
+  {
+   "source": [
+    "#### Read dataset"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = spark.read.format(\"csv\")\\\n",
+    "  .option(\"header\", True)\\\n",
+    "  .option(\"inferSchema\", True)\\\n",
+    "  .load(\"wasbs://publicwasb@mmlspark.blob.core.windows.net/company_bankruptcy_prediction_data.csv\")\n",
+    "# print dataset size\n",
+    "print(\"records read: \" + str(df.count()))\n",
+    "print(\"Schema: \")\n",
+    "df.printSchema()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "display(df)"
+   ]
+  },
+  {
+   "source": [
+    "#### Split the dataset into train and test"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train, test = df.randomSplit([0.85, 0.15], seed=1)"
+   ]
+  },
+  {
+   "source": [
+    "#### Add featurizer to convert features to vector"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyspark.ml.feature import VectorAssembler\n",
+    "feature_cols = df.columns[1:]\n",
+    "featurizer = VectorAssembler(\n",
+    "    inputCols=feature_cols,\n",
+    "    outputCol='features'\n",
+    ")\n",
+    "train_data = featurizer.transform(train)['Bankrupt?', 'features']\n",
+    "test_data = featurizer.transform(test)['Bankrupt?', 'features']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_data.show(10)"
+   ]
+  },
+  {
+   "source": [
+    "#### Check if the data is unbalanced"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_data.groupBy(\"Bankrupt?\").count().show()"
+   ]
+  },
+  {
+   "source": [
+    "#### Model Training"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mmlspark.lightgbm import LightGBMClassifier\n",
+    "model = LightGBMClassifier(objective=\"binary\", featuresCol=\"features\", labelCol=\"Bankrupt?\", isUnbalance=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = model.fit(train_data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mmlspark.lightgbm import LightGBMClassificationModel\n",
+    "model.saveNativeModel(\"/lgbmcmodel\")\n",
```

Comment:
perhaps we can call this lgbmclassifier.model and add a cried markdown description that this allows you to extract the underlying lightGBM model for fast deployment after you train on spark